package.json

```diff
@@ -29,7 +27,8 @@
   "homepage": "https://github.com/rdkcentral/firebolt-core-sdk/blob/main/src/modules//ottx/firebolt-openrpc#readme",
   "devDependencies": {
     "jest": "^27.3.1",
-    "webpack-cli": "^4.8.0"
+    "webpack-cli": "^4.8.0",
+    "@firebolt-js/schemas": "../firebolt-schemas"
```

Comment:
why was this added? This would be a cyclical dependency since schemas depends on openrpc

Reply:
Hmm, you're right. I used the shared `ProviderResponse` schema in my test module `provider.json`

Open to ideas here...

Reply:
They're both "dev dependencies" so it's not actually a cyclical dependency for the artifacts we build, but I agree it points to some sort of issue worth figuring out.